style.css

```diff
@@ -73,4 +73,8 @@ form > div > button {
 form > div > button{
```

Comment:
use classes for styling

Reply:
Used direct selectors in those places where creating classes is unnecessary.